packages/algolia/test/client.test.ts

```diff
@@ -153,6 +159,22 @@ describe('Algolia Search Client', () => {
     });
   });
 
+  test('Should search event entity', async () => {
```

Comment:
I think this test title should probally say if its supposed to be in the past or future